setup.py

```diff
@@ -7,24 +7,32 @@
 TEST_DEPS = ["coverage", "pytest", "pytest-cov"]
 DOCS_DEPS = ["sphinx", "sphinx-rtd-theme", "sphinx-autoapi", "recommonmark"]
 CHECK_DEPS = ["isort", "flake8", "flake8-quotes", "pep8-naming", "mypy", "black"]
-REQUIREMENTS = ["invenia-cli-utils"]
-
 EXTRAS = {
     "test": TEST_DEPS,
     "docs": DOCS_DEPS,
     "check": CHECK_DEPS,
     "dev": TEST_DEPS + DOCS_DEPS + CHECK_DEPS,
 }
 
+
+REQUIREMENTS = [
+    "invenia-cli-utils",
```

Comment:
Why is this required?

Reply:
Looks like this is a bug in our template creator. This isn't needed for github projects